contracts/Escrow.sol

```diff
@@ -40,14 +40,20 @@ contract Escrow {
 
     mapping(address => TrustedHandler) public trustedHandlers;
 
-    constructor(address _eip20, address _canceler, uint256 _expiration) public {
+    constructor(
+        address _eip20,
+        address _canceler,
+        uint256 _expiration,
+        address[] _handlers
```

Comment:
This means we'll need to update all the contracts, correct?

Reply:
yes